.changeset/tender-parrots-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-module-addons-contrib': patch
+---
+
+Refactored Report issue body in the tech-doc addons by getting the app title from `appconfig.yml` using `configApiRef`, In case `appTitle` not mentioned app Tile `new const` will default to `Backstage`
```

Comment:
This will be included in the list of changes for the next release, so I think we can be a bit more concise and omit some details. Here's a suggestion:

Use `app.title` from `app-config.yaml` when creating new Documentation Feedback issue. `Backstage` is the default value.

Reply:
Accept @skgandikota  , rephrased as mentioned !   Thanks for sharing suggestion